function_calling/notebooks/function_calling_guide.ipynb

```diff
@@ -9,7 +9,7 @@
   },
```

Comment:
This line in the notebook should have a parenthesis at the end, otherwise it throws an error.

`raise ValueError(f'Invalid message type: {msg.type}')`

Reply:
fixed

Reply:
fixed